pulp_rpm/app/downloaders.py

```diff
@@ -26,6 +28,18 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
 
+class NoUlnCredentials(Exception):
```

Comment:
In case this is also an error, you can add the suffix `Error`.

```suggestion
class NoUlnCredentialsError(Exception):
```

Reply:
Done